state_machine/drivers/pycubedmini/lib/pycubed.py

```diff
@@ -18,21 +19,182 @@
 import pwmio
 import bmx160
 import drv8830
-from os import listdir, stat, statvfs, mkdir
 from bitflags import bitFlag, multiBitFlag
 from micropython import const
 import adafruit_tsl2561
 
+
+"""
+IMU-related functions
+"""
+
+def acceleration():
```

Comment:
I'd really like to have units in the docstrings for the IMU readings

Reply:
Will do